### ISSUE TITLE
Super Struct

### ISSUE THREAD

**Issue by trixr4kdz** [kdahlquist/GRNmap#69]
This is implemented in the super-struct branch. Currently a super struct stores a copy of the GRNstruct for the original run. However, we realize that during a run we output a .mat and excel file for the input file for that run. Would a super-struct be redundant?

**Comment by bengfitzpatrick** [comment]
Not sure what to say without more detail. What do you mean by "original run" -- is the code being run multiple times?

**Comment by juansc** [comment]
To clarify. When we run GRNmodel for the first time, we create a variable called GRNSuperStruct. After calling readInputSheet, lse, and output, the resulting GRNstruct is appended to the GRNSuperStruct. If we run GRNmodel again without using `clear` or `clear all` the GRNSuperStruct persists, and so the second time around after calling readInputSheet, lse, and output, the new GRNstruct is also appened to the GRNSuperStruct. This was proposed so that we may have some way of comparing data across multiple runs.

**Comment by bengfitzpatrick** [comment]
I would like to postpone work on this issue till we can get together, which probably means the week of 25 May. Juan, I realize you'll not be around for it, so let's make sure Trixie is comfortable with matters.

**Comment by kdahlquist** [comment]
Did we figure out how to initialize all the arrays to zero?  We need to know the answer to this before we can proceed with this issue.

**Comment by trixr4kdz** [comment]
The super-struct branch has been deleted as per issue #164 as part of the branch clean-up so this issue seems stale. The reasoning behind the deletion is that the super-struct branch was falling behind the rest of the branches.

**Comment by kdahlquist** [comment]
as per @trixr4kdz 's recommendation, we are closing this one.
